templates/galaxy/config/job_conf.yml

```diff
@@ -175,6 +175,17 @@ galaxy_jobconf:
         amqp_consumer_timeout: 2.0
         amqp_publish_retry: "true"
         amqp_publish_retry_max_retries: 60
+    - id: pulsar_eu_cz02
+      load: galaxy.jobs.runners.pulsar:PulsarMQJobRunner
+      params:
+        amqp_url: "pyamqp://galaxy_cz01:{{ rabbitmq_password_galaxy_cz01 }}@mq.galaxyproject.eu:5671//pulsar/galaxy_cz01?ssl=1"
```

Comment:
```suggestion
        amqp_url: "pyamqp://galaxy_cz02:{{ rabbitmq_password_galaxy_cz02 }}@mq.galaxyproject.eu:5671//pulsar/galaxy_cz02?ssl=1"
```

Reply:
Once this is committed, I can add the relevant changes to our RabbitMQ playbook

Reply:
Is it really necessary? Cause it's the very same pulsar here, only manager differs, and by design, it's not possible to have two different AMQP connections defined in one pulsar. I would need to create another one for the second connection.

Reply:
If it is the same Pulsar process, then not, sorry didn't see that